unfetter-discover/unfetter-ui
# Issue 1034 (Prev/Next Group buttons for baselines wizard)

Fixes unfetter-discover/unfetter#1034

Previous and Next Group buttons are introduced to allow the user to skip to the capability selection steps for each capability group.  This allows for a more rapid capability selection phase ahead of applying weights to the capabilities themselves.

## To test:
1. Pull this PR
2. Restart unfetter-ui:  `docker-compose restart unfetter-ui`
3. Start a new baseline
4. Select some capability groups
5. Click Continue
6. Select some capabilities
7. Click `Next Group` to skip over selected capabilities and onto the next group in the wizard
8. Click `Previous Group` to go back to the previous group
9. `Previous Group` and `Next Group` should only be visible when appropriate
